ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -22,21 +22,38 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
     Ok(views.html.app("Recipeasy"))
   }
 
-  def curateRecipePage = Action { implicit request =>
+  def curateNewRecipe = Action { implicit request =>
     val newRecipe = db.getNewRecipe
     newRecipe match {
+      case Some(r) => Redirect(routes.Application.curateRecipe(r.id))
+      case None => NotFound
+    }
+  }
+
+  def curateRecipe(id: String) = Action { implicit request =>
+    val recipe = db.getRecipe(id)
+    view(recipe, editable = true)
+  }
+
+  def viewRecipe(id: String) = Action { implicit request =>
+    val recipe = db.getRecipe(id)
```

Comment:
@tsop14 the code only retrieve am `id` there which is then use top retrieve the `curated_recipe` in `view`:
```scala
val curatedRecipe = CuratedRecipe.fromRecipe(r)
```
So I think this code will retrieve all the saved content of the `curated_recipe` table